Microsoft.Research/RegressionTest/ClousotTests/ClousotTests.csproj

```diff
@@ -179,6 +179,12 @@
     <None Include="..\TestFrameworkOOB\UserFeedback.cs">
       <Link>Sources\UserFeedback.cs</Link>
     </None>
+    <Compile Include="..\IteratorAnalysis\IteratorSimpleContract\IteratorSimpleContract.cs">
+      <Link>Sources\IteratorSimpleContract.cs</Link>
+    </Compile>
+    <Compile Include="..\TestPostconditionInference\Filtering.cs">
+      <Link>Sources\Filtering.cs</Link>
```

Comment:
This file causes ClousotTests not to show in Test Explorer, because it contains methods declared `extern`.

    Error loading C:\…\CodeContracts\Microsoft.Research\RegressionTest\ClousotTests\bin\Debug\ClousotTests.dll: Unable to load the test container 'C:\…\CodeContracts\Microsoft.Research\RegressionTest\ClousotTests\bin\Debug\ClousotTests.dll' or one of its dependencies. Error details:
    System.TypeLoadException: Could not load type 'TestPostconditionInference.FilterRedundantPostcondition' from assembly 'ClousotTests, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null' because the method 'GetApplicationVersion' has no implementation (no RVA).
    System.TypeLoadException: Could not load type 'TestPostconditionInference.Array' from assembly 'ClousotTests, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null' because the method 'get_Length' has no implementation (no RVA).